.github/workflows/main.yml

```diff
@@ -157,16 +185,6 @@ jobs:
         if: ${{ always() }}
         run: moodle-plugin-ci phplint
 
-      - name: PHP Copy/Paste Detector
-        continue-on-error: true # This step will show errors but will not fail
-        if: ${{ always() }}
-        run: moodle-plugin-ci phpcpd
-
-      - name: PHP Mess Detector
```

Comment:
PHP Mess Detector continues to be supported in Moodle Plugin CI. Could you share the reasoning behind your decision to remove it, @lucaboesch?

Reply:
I must have confused it with phpcpd. Sorry.
It is back in.